.gitignore

```diff
@@ -13,3 +13,5 @@
 /coverage
 
 !/docker-npm
+
+/.gitlab-ci-local/
```

Comment:
Wouldn't we like to know, if the tool by accident creates a .gitlab-ci-local folder ?

Reply:
I would not check this via git because it might be that someone will commit a .gitlab-ci-local folder which is not what we want I guess. 

So we could create a test to check if a .gitlab-ci-local folder was created or add it in the package.json file in the end of the test command.

Reply:
Yah, you are right...